ui/app/controllers/vault/cluster/access/mfa/methods/create.js

```diff
@@ -95,7 +96,7 @@ export default class MfaMethodCreateController extends Controller {
         // first save method
         yield this.method.save();
         if (this.enforcement) {
-          this.enforcement.mfa_methods.addObject(this.method);
+          this.enforcement.mfa_methods = addToArray(this.enforcement.mfa_methods.slice(), this.method);
```

Comment:
`this.enforcement.mfa_methods` is not an Array, so `slice` changes it to an array before passing it into `addToArray` because non-arrays can't be spread. I suppose I could update the `addToArray` method to use slice instead of spread. 

One thing I like about this approach, is it helps us know what types we're working with and keep awareness that `mfa_methods` is not a pure array, for example.

Reply:
thank you for explaining!